### PR TITLE
chore: Migrate from Firestore to JSON data (temp. fix)

### PR DESCRIPTION
**Pull request by CIPHERTron**

### CODE REVIEW

File: client/src/views/Home.tsx [hunk 1]
```diff
@@ -1,143 +1,158 @@
...
 
 // Libraries
-import { Box, makeStyles } from '@material-ui/core';
-import { Link } from 'react-router-dom';
-import firebase from 'firebase/app';
-import 'firebase/firestore';
+import { Box, makeStyles } from "@material-ui/core"
+import { Link } from "react-router-dom"
+// import firebase from "firebase/app"
+import "firebase/firestore"
```
Comment (abhibhaw) [inline client/src/views/Home.tsx:7]: @CIPHERTron are we still using this?

**Review by abhibhaw — APPROVED** [review]
@CIPHERTron time to configure styling standards for this project as well 🚀

File: client/src/views/Home.tsx (hunk 1 above)
Comment (CIPHERTron) [reply client/src/views/Home.tsx:7]: We might, afterwards :')

**Comment by CIPHERTron** [comment]
> @CIPHERTron time to configure styling standards for this project as well

Retire this project, Daksh Sir will make design. Then, we'll make another project with awesome UI (* -- *)

**Pull request merged**